landmark_notes.md

```diff
@@ -0,0 +1,404 @@
+## Sun
```

Comment:
Good call on the notes. These are great to have as reference for the descriptions in case we ever want to update them or try to remember what we chose to include.